site/content/getting-started.md

```diff
@@ -96,6 +96,14 @@ mvn spring-boot:run -Drun.arguments=--hono.client.host=localhost,--hono.client.u
 Replace *localhost* with the name or IP address of the host that Docker is running on.
 {{% /warning %}}
 
+If the command line client is behind a proxy, you can use the proxy options to connect. The default proxy type is SOCKS5. Run the client from the `example` folder as follows:
+                                                                                                                
+~~~sh
+mvn spring-boot:run -Drun.arguments=--hono.client.host=localhost,--hono.client.username=consumer@HONO,--hono.client.password=verysecret,--hono.client.proxyHost=proxy.company.com,--hono.client.proxyPort=8080,--hono.client.proxyUsername=my-company-id,--hono.client.proxyPassword=verysecret,--hono.client.proxyType=SOCKS4
```

Comment:
sure you want to promote `SOCKS4` instead of `SOCKS5`? Also, the port (8080) is quite unusual for a socks proxy and might confuse people. On the other hand you may want to give an example of how to (ab)use an HTTPS proxy for an AMQPS connection ;-)

Reply:
As default was SOCKS5, I wanted to show how one can configure otherwise. On omitting this parameter, SOCKS5 is set by default. Anyways, it doesn't hurt setting default again. :-)

Yes port 8080 was misleading. I will change it.